app/presenters/email_alert_presenter.rb

```diff
@@ -24,6 +25,19 @@ def body
     end
   end
 
+  def tags
+    { format: document.publishing_api_document_type }.deep_merge(metadata)
+  end
+
+  def metadata
```

Comment:
You're both right!  Pulling them from the objects looks to be the way it's got to be done, but it is quite a hard to parse method.  @christophwong could you spend a short time seeing if it can be made clearer?